### ISSUE TITLE
SpaceAroundOperator: false positive with captured funs built from unquoted fragments

### ISSUE THREAD

**Issue by NobbZ** [rrrene/credo#481]
### Environment

* Credo version (`mix credo -v`): 0.8.10
* Erlang/Elixir version (`elixir -v`):
```
Erlang/OTP 20 [erts-9.1] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:10] [hipe]

Elixir 1.5.2
```
* Operating system: Funtoo Linux

### What were you trying to do?

```ex
{time, r} = :timer.tc(&unquote(module).unquote(part)/0)
```

### Expected outcome

No warning

### Actual outcome

```plain
┃ [C] ↗ There are spaces around operators most of the time, but not here.
```

**Comment by rrrene** [comment]
Thanks for reporting! :+1:

**Comment by rrrene** [comment]
@NobbZ Thx for reporting this and sorry for the long delay! :+1: It should be be fixed in the latest version of Credo.

If it is not, please feel free to re-open this issue!
